src/standup/interfaces/Standup.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface IStandup {
```

Comment:
Is this the recommended way to do interface naming in NestJS? From typescript no `I` prepending is needed https://www.typescriptlang.org/docs/handbook/2/objects.html

Reply:
To be honest I don't really know.. A colleague at work always did this, so I adapted my style to that.
I could change this if its not necessary.

Reply:
I like to keep things vanilla / recommended from official docs, because it makes it easier to onboard new people. If you could please update 👍